modules/caption-impl/src/main/java/org/opencastproject/caption/converters/GoogleSpeechCaptionConverter.java

```diff
@@ -83,65 +83,67 @@ public List<Caption> importCaption(InputStream inputStream, String languageLineS
       for (int i = 0; i < resultsArray.size(); i++) {
         JSONObject resultElement = (JSONObject) resultsArray.get(i);
         JSONArray alternativesArray = (JSONArray) resultElement.get("alternatives");
-        if (alternativesArray != null && alternativesArray.size() > 0) {
-          JSONObject alternativeElement = (JSONObject) alternativesArray.get(0);
-          // remove trailing space in order to have correct transcript length
-          String transcript = ((String) alternativeElement.get("transcript")).trim();
-          if (transcript != null) {
-            JSONArray timestampsArray = (JSONArray) alternativeElement.get("words");
-            if (timestampsArray == null || timestampsArray.isEmpty()) {
-              logger.warn("Could not build caption object for job {}, result index {}: timestamp data not found",
-                      jobId, i);
-              continue;
-            }
-            // Force a maximum line size of transcriptionLineSize + one word
-            String[] words = transcript.split("\\s+");
-            StringBuffer line = new StringBuffer();
-            int indexFirst = -1;
-            int indexLast = -1;
-            for (int j = 0; j < words.length; j++) {
-              if (indexFirst == -1) {
-                indexFirst = j;
-              }
-              line.append(words[j]);
-              line.append(" ");
-              if (line.length() >= transcriptionLineSize || j == words.length - 1) {
-                indexLast = j;
-                // Create a caption
-                double start = -1;
-                double end = -1;
-                if (indexLast < timestampsArray.size()) {
-                  // Get start time of first element
-                  JSONObject wordTSList = (JSONObject) timestampsArray.get(indexFirst);
-                  if (wordTSList.size() == 3) {
-                    // Remove 's' at the end
-                    Number startNumber = NumberFormat.getInstance(Locale.US).parse(removeEndCharacter((wordTSList.get("startTime").toString()), "s"));
-                    start = startNumber.doubleValue();
+            if (alternativesArray != null && alternativesArray.size() > 0) {
+              JSONObject alternativeElement = (JSONObject) alternativesArray.get(0);
+              if (!alternativeElement.isEmpty()) {
```

Comment:
This starts a block of code, so stuff below this should be indented another level.